test/functional/glance_controller_test.go

```diff
@@ -190,6 +201,10 @@ var _ = Describe("Glance controller", func() {
 				),
 			)
 			mariadb.SimulateMariaDBDatabaseCompleted(glanceTest.Instance)
```

Comment:
Is `mariadb.SimulateMariaDBDatabaseCompleted` required as long as we're switching to `mariadb.SimulateMariaDBAccountCompleted` ? (The same comment applies to all the occurrences).

Reply:
I think we need both for now.